src/models/mcawsModels.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const Sequelize = require('sequelize');
+
+function getDbOrmObj(user, passwd, host, dbname) {
```

Comment:
Where are we handling the errors(e.g. wrong password/user or database connection issue)

Reply:
handle in index_updatedDB.js

Reply:
I do not see it. Can you please point me to the code?

Reply:
index_update_db.js
line number 41 and 48